cibuild.sh

```diff
@@ -7,14 +7,19 @@ usage()
     echo "usage: cibuild.sh [options]"
     echo ""
     echo "Options"
-    echo "  --debug     Build Debug (default)"
-    echo "  --release   Build Release"
-    echo "  --nocache   Force download of toolsets"
-
+    echo "  --debug               Build Debug (default)"
+    echo "  --release             Build Release"
+    echo "  --skiptests           Do not run tests"
+    echo "  --skipcrossgen        Do not crossgen the bootstrapped compiler"
+    echo "  --skipcommitprinting  Do not print commit information"
```

Comment:
I can't remember the reason we did the commit printing in the first place.  @agocke may remember.